### PR TITLE
Configure Sentry exceptions reporting

### LINKED ISSUE #411: Add Sentry for errors tracking
```sh
yarn add @sentry/react @sentry/tracing
```

```jsx
import * as Sentry from "@sentry/react";
import { BrowserTracing } from "@sentry/tracing";


Sentry.init({
  dsn: "https://98376d41178a4faa8d42c4d5c17b146d@o227269.ingest.sentry.io/6763898",
  integrations: [new BrowserTracing()],

  // Set tracesSampleRate to 1.0 to capture 100%
  // of transactions for performance monitoring.
  // We recommend adjusting this value in production
  tracesSampleRate: 1.0,
});
```

### PR DESCRIPTION
**Pull request by louisinger**
* install sentry/browser pkg
* call Sentry.init in index.tsx
* current DSN 'https://2b7688ea928a414d8619f11e998a0aa3@app.glitchtip.com/1844' is a personnal one from [glitchtip.com](https://glitchtip.com/). We may want a global one accessible from all of us ? cc @tiero ?
* Sentry only enabled in case of `process.env === 'production'`

it closes #411 

@tiero please review

### CODE REVIEW

File: src/presentation/index.tsx [hunk 1]
```diff
@@ -5,10 +5,17 @@ import { PersistGate } from 'redux-persist/integration/react';
...
 import { persistor } from '../application/redux/store';
 import App from './app';
+import * as Sentry from '@sentry/browser';
 
 import './styles/index.css';
 import './styles/fonts.css';
 
+if (process.env.NODE_ENV === 'production') {
+  Sentry.init({
+    dsn: 'https://2b7688ea928a414d8619f11e998a0aa3@app.glitchtip.com/1844',
```
Comment (tiero) [inline src/presentation/index.tsx:15]:
```suggestion
    dsn: 'https://4a036f974898ba84cb62e04402fec31c@o227269.ingest.sentry.io/4506381926531072',
```

**Comment by tiero** [comment]
Any reason to not use the React package of Sentry?

```js
//...
import * as Sentry from "@sentry/react";

Sentry.init({
  dsn: "https://4a036f974898ba84cb62e04402fec31c@o227269.ingest.sentry.io/4506381926531072",
  integrations: [
    new Sentry.BrowserTracing({
      // Set 'tracePropagationTargets' to control for which URLs distributed tracing should be enabled
      tracePropagationTargets: ["localhost", /^https:\/\/yourserver\.io\/api/],
    }),
    new Sentry.Replay(),
  ],
  // Performance Monitoring
  tracesSampleRate: 1.0, //  Capture 100% of the transactions
  // Session Replay
  replaysSessionSampleRate: 0.1, // This sets the sample rate at 10%. You may want to change it to 100% while in development and then sample at a lower rate in production.
  replaysOnErrorSampleRate: 1.0, // If you're not already sampling the entire session, change the sample rate to 100% when sampling sessions where errors occur.
});

const container = document.getElementById(“app”);
const root = createRoot(container);
root.render(<App />);
```

**Comment by louisinger** [comment]
> Any reason to not use the React package of Sentry? @louisinger 

It looks like a wrapper or the Sentry class, not sure the react components add a real value 🤔

**Comment by tiero** [comment]
Solve conflicts

**Pull request merged**